### DISCUSSION TITLE
Github Copilot facing activation issues with student mail.

### DISCUSSION THREAD

**Discussion by Vaishnav-Varma** [community/community#31195] [Copilot Conversations]
I gained early access to Github Copilot for VS Code as a student back in November 2021, and now the access has expired. 
I was notified that my student account has free access to Copilot, so I reinstalled the VS Code extension and tried to activate it through my student mail, but it isn't activating.
How do I resolve this issue?
![image](https://user-images.githubusercontent.com/89126788/187129163-e5d8e701-c65c-4c93-848a-4f3b6ddf6df1.png)

**Comment by davecheney** [comment]
@Vaishnav-Varma I can confirm your student access to copilot is active. 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by Vaishnav-Varma** [reply]
I have to sign out on VS Code right? I'm on a Windows machine.

**Reply by Vaishnav-Varma** [reply]
I signed out, but how do I make it so that all the previous extensions I had installed for VS Code remain?

**Reply by davecheney** [reply]
Signing out does not uninstall extensions.
